pkg/apis/rook.io/v1alpha2/storage.go

```diff
@@ -97,7 +97,6 @@ func (s *StorageScopeSpec) resolveNodeSelection(node *Node) {
 }
 
 func (s *StorageScopeSpec) resolveNodeConfig(node *Node) {
-	resolveString(&(node.Location), s.Location, "")
```

Comment:
Why is this removed?

Reply:
This is the part that's being ignored. This is pulling in the cluster level Location of the StorageScopeSpec.

Reply:
So we will only look at the `location` added at a lower scope such as individual nodes? I don't know why anyone would even specify the cluster-level location though, so it makes sense.